python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -1364,6 +1364,74 @@ private module PrivateDjango {
 
           override string getMimetypeDefault() { result = instance.getMimetypeDefault() }
         }
+
+        /**
+         * A call to `set_cookie` on a HTTP Response.
+         */
+        class DjangoResponseSetCookieCall extends HTTP::Server::CookieWrite::Range,
+          DataFlow::MethodCallNode {
+          DjangoResponseSetCookieCall() {
+            this.calls(django::http::response::HttpResponse::instance(), "set_cookie")
+          }
```

Comment:
I wonder if, in the long run, we want `instance` to return an API node, so we can do without `MethodCallNode` here...

Reply:
Yes, once we can do this, it would be really nice :+1: